### DISCUSSION TITLE
Adding exam exercises

### DISCUSSION THREAD

**Discussion by pandaero** [JCluzet/42_EXAM#22] [General]
Is this something that is simple to do? Could we have some guidance in order to fork and add them, then we can keep the exercises up to date (several missing from rank 2 level 4)

**Comment by pandaero** [comment]
I just forked the repo and tried to add an exercise, then I tested it, and it worked great! I'll try to add more and then pull request 😄

**Reply by pandaero** [reply]
check the repository [here](https://github.com/pandaero/42_EXAM), I will be working directly on the main branch

**Comment by JCluzet** [comment]
Hi pandaero 👋

Thanks for your interest in 42_EXAM and your contribution.

I have just added a [small documentation](https://github.com/JCluzet/42_EXAM/blob/main/CONTRIBUTING.md) on how to contribute by adding exercises.
If you have any question don't hesitate :)
